src/views/DataStream/FormFields/FormFieldsDataStream.vue

```diff
@@ -1146,6 +1168,8 @@
   // Using the store
   const store = useAccountStore()
 
+  const hasPermissionDataStream = computed(() => store.hasPermissionToEditDataStream)
```

Comment:
Since we usually have 2 permissions (view and edit), think we should be a bit more explicit over here
```suggestion
  const hasPermissionToEditDataStream = computed(() => store.hasPermissionToEditDataStream)
```

Reply:
done